src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -52,7 +52,7 @@ trait ClusterComponent extends LeoComponent {
   class ClusterTable(tag: Tag) extends Table[ClusterRecord](tag, "CLUSTER") {
     def id =                          column[Long]              ("id",                    O.PrimaryKey, O.AutoInc)
     def clusterName =                 column[String]            ("clusterName",           O.Length(254))
-    def googleId =                    column[UUID]              ("googleId",              O.Unique)
```

Comment:
Oh interesting, before `(id, googleId)` was a composite primary key in the DB but this was not represented in slick -- slick had only `id` as the PK. I'm glad this is fixed now.